examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -1648,6 +1666,7 @@ endpoint 0 {
 
   server cluster Groups {
     ram      attribute nameSupport;
+    persist  attribute featureMap;
```

Comment:
new data seems to be persist (i.e. stored in NVM).

I believe they should be 'ram' instead as we should not store these in flash. realistically they should not e modifiable at all. Not sure if it makes a difference, however I would go based on existing values which are ram